.travis.yml

```diff
@@ -33,8 +33,7 @@ addons:
 script: ./travis.sh 
 
 notifications:
-  email:
-    - cukes-devs@googlegroups.com
+  email: false
```

Comment:
I meant sending mails to the commit author and committer (that I agree is useful).

I'd rather have both CI configured in the same way, or committers would have to check in two places (or more likely ignore Windows failures).